serialization/compact_config.go

```diff
@@ -51,3 +63,55 @@ func (cc *CompactConfig) ensureSerializers() {
 		cc.serializers = make(map[string]CompactSerializer)
 	}
 }
+
+func (cc *CompactConfig) checkNoDefaultSerializer() error {
+	// note that the check we are doing is just information.
+	// the default serializers are looked uo before the compact serializer,
+	// so the user cannot override any default serializer.
+	var cr CompactReader
+	var cw CompactWriter
```

Comment:
why is these two here?

Reply:
They are not necessary, removed at: https://github.com/hazelcast/hazelcast-go-client/pull/942/commits/5544c931150397ebc71360304bd91933a0b5bb87